addon/mixins/responds-to-print.js

```diff
@@ -0,0 +1,35 @@
+import Ember from 'ember';
+
+// Triggers 'print' event and calls 'print' handler when the page is being printed.
+// Browser support: http://caniuse.com/#feat=matchmedia
+export default Ember.Mixin.create(Ember.Evented, {
+
+  print: Ember.$.noop,
+
+  didInsertElement: function () {
+    this._super();
+
+    this._RespondsToPrint_mediaQueryList = window.matchMedia('print');
```

Comment:
`window.matchMedia('print')` would be a fatal error in IE9, right? Ember dropped IE8 support but not yet IE9, AFAIK. Still, I'm ok with it. F*** IE. Two major versions (10, 11) is getting to be a conventional standard for support.